src/testing/Testing.hpp

```diff
@@ -82,6 +82,23 @@ boost::test_tools::predicate_result equals(const Eigen::MatrixBase<DerivedA> &A,
   return true;
 }
 
+/// equals to be used in tests. Prints both operatorans on failure
+template <typename TA, typename TB>
+boost::test_tools::predicate_result equals(const std::vector<TA> &A,
+                                           const std::vector<TB> &B,
+                                           double                 tolerance = math::NUMERICAL_ZERO_DIFFERENCE)
+{
+  Eigen::MatrixXd MatrixA(A.size(), 1);
```

Comment:
What happens if the two input container do not have the same size? I would propose to assert it here and exit immediately as they are not the same(?)

Reply:
Will do this.